src/Iot.Device.Bindings/Iot.Device.Bindings.csproj

```diff
@@ -11,6 +11,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <!-- Excluding RGBLedMatrix for now till we figure if it is ok to add dependency on stuff like System.Drawing  -->
+    <_ExcludeFromCompile Include="$(DeviceRoot)/RGBLedMatrix/**/*.cs" />
```

Comment:
@joperezr I am excluding this project for know till I know if it will be ok to add more dependencies on System.Darwing. Please advise if you have any insight about that.

Reply:
@joperezr I have included and added more dependencies (e.g. System.Drawing). let's talk when you are back